### DISCUSSION TITLE
query works but not with select count(*) from stream

### DISCUSSION THREAD

**Discussion by bif-bof** [datalust/seq-tickets#1258] [Help]
Exception.Message = 'Auf das verworfene Objekt kann nicht zugegriffen werden.'
=> this query shows results

select count(*) from stream where Exception.Message = 'Auf das verworfene Objekt kann nicht zugegriffen werden.'
=> this query doesn't
![image](https://user-images.githubusercontent.com/75050180/125265065-c0dcc400-e304-11eb-90af-db478bde6b7a.png)

I wonder why this is not working - Am I doing something wrong. I use select count(*) from stream very often

**Comment by bif-bof** [comment]
Strange. After closing the SEQ GUI and using the query again - it work's

**Reply by nblumhardt** [reply]
Thanks for the follow-up. This will be because the display selector "remembers" being set to "chart" - pressing "table" should again show the results:

![image](https://user-images.githubusercontent.com/342712/125363272-add4fd00-e3b3-11eb-9fcf-72ec66df190b.png)

(We'll give some thought to improving UX here :-))
